streams/test-utils/src/main/java/org/apache/kafka/streams/TopologyTestDriver.java

```diff
@@ -395,7 +395,7 @@ public void pipeInput(final ConsumerRecord<byte[], byte[]> consumerRecord) {
                 consumerRecord.value())));
 
             // Process the record ...
-            ((InternalProcessorContext) task.context()).setRecordContext(new ProcessorRecordContext(consumerRecord.timestamp(), offset, topicPartition.partition(), topicName));
+            ((InternalProcessorContext) task.context()).setRecordContext(new ProcessorRecordContext(consumerRecord.timestamp(), offset, topicPartition.partition(), topicName, null));
```

Comment:
Why pass in null than `consumerRecord.headers()`??

Reply:
Ack.